src/main/java/com/zaxxer/influx4j/Point.java

```diff
@@ -896,15 +896,25 @@ private static void escapeCommaEqualSpace(final String string, final ByteBuffer
       }
    }
 
-   private static void escapeDoubleQuote(final String string, final ByteBuffer buffer) {
-      if (string.indexOf('"') != -1) {
+   private static void escapeDoubleQuoteAndBackslashIfLast(final String string, final ByteBuffer buffer) {
+      final int lastCharacterIndex = string.length() - 1;
```

Comment:
This looks like it might break support for multi-byte characters, since the string.length will not be equal to the bytes.length.

Reply:
Fixed.